.husky/pre-push

```diff
@@ -0,0 +1 @@
+pnpm run typecheck; pnpm run lint;
```

Comment:
think adding a build here is too much?

Reply:
TS and linter errors are blockers for build, since if they succeed, build will succeed - in this case we could save from build step, wdyt?